{"uid":"d317f8a5d2f7d7ad","name":"test_upgrade_from_stable","fullName":"tests.integration.high_availability.test_upgrade_from_stable#test_upgrade_from_stable","historyId":"fe154b88a315bf4fc4617f727e70d7c2","time":{"start":1722562847706,"stop":1722563748391,"duration":900685},"description":"Test updating from stable channel.","descriptionHtml":"<p>Test updating from stable channel.</p>\n","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 129, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1792, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/tests/integration/high_availability/test_upgrade_from_stable.py\", line 114, in test_upgrade_from_stable\n    await ops_test.model.block_until(\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1045, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1722562670729,"stop":1722562670729,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1722562670729,"stop":1722562670729,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1722562670729,"stop":1722562670997,"duration":268},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1722562847706,"stop":1722562847706,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test updating from stable channel.","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 129, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1792, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/tests/integration/high_availability/test_upgrade_from_stable.py\", line 114, in test_upgrade_from_stable\n    await ops_test.model.block_until(\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1045, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/mysql-k8s-operator/mysql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","steps":[],"attachments":[{"uid":"c9d790e806ac8d28","name":"log","source":"c9d790e806ac8d28.txt","type":"text/plain","size":1601}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1722563748396,"stop":1722563748396,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1722563748397,"stop":1722563748845,"duration":448},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1722563748846,"stop":1722563748846,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1722563748846,"stop":1722563748846,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1722563748846,"stop":1722563748846,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"@pytest.mark.skipif(False, reason='Requires amd64 architecture')"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.high_availability"},{"name":"suite","value":"test_upgrade_from_stable"},{"name":"host","value":"fv-az658-492"},{"name":"thread","value":"12488-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.high_availability.test_upgrade_from_stable"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"cd0e25a345083db","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":6,"skipped":40,"passed":132,"unknown":0,"total":178},"items":[{"uid":"c098e0e9fef21a83","reportUrl":"../2134//#testresult/c098e0e9fef21a83","status":"passed","time":{"start":1722476192145,"stop":1722476400765,"duration":208620}},{"uid":"93d4e7e18c882b97","reportUrl":"../2128//#testresult/93d4e7e18c882b97","status":"passed","time":{"start":1722389672247,"stop":1722389877989,"duration":205742}},{"uid":"e535939fa8028040","reportUrl":"../2126//#testresult/e535939fa8028040","status":"passed","time":{"start":1722303364160,"stop":1722303542847,"duration":178687}},{"uid":"476a02b8addf6084","reportUrl":"../2121//#testresult/476a02b8addf6084","status":"passed","time":{"start":1722217006851,"stop":1722217216776,"duration":209925}},{"uid":"5c272b43913fc6ef","reportUrl":"../2120//#testresult/5c272b43913fc6ef","status":"passed","time":{"start":1722130689611,"stop":1722130877481,"duration":187870}},{"uid":"992eef6131e177bf","reportUrl":"../2119//#testresult/992eef6131e177bf","status":"passed","time":{"start":1722044132362,"stop":1722044319763,"duration":187401}},{"uid":"36b82e53c774ce24","reportUrl":"../2112//#testresult/36b82e53c774ce24","status":"passed","time":{"start":1721957770187,"stop":1721957955309,"duration":185122}},{"uid":"4dd1a701085d1d0d","reportUrl":"../2105//#testresult/4dd1a701085d1d0d","status":"passed","time":{"start":1721871331126,"stop":1721871521972,"duration":190846}},{"uid":"9c306b44c78c6106","reportUrl":"../2103//#testresult/9c306b44c78c6106","status":"passed","time":{"start":1721785411182,"stop":1721785579556,"duration":168374}},{"uid":"5bd9bd286b15e0f1","reportUrl":"../2101//#testresult/5bd9bd286b15e0f1","status":"passed","time":{"start":1721698975602,"stop":1721699181266,"duration":205664}},{"uid":"d822c3611284ed81","reportUrl":"../2095//#testresult/d822c3611284ed81","status":"passed","time":{"start":1721612191122,"stop":1721612379131,"duration":188009}},{"uid":"d32d40ca24475271","reportUrl":"../2094//#testresult/d32d40ca24475271","status":"passed","time":{"start":1721525688697,"stop":1721525867037,"duration":178340}},{"uid":"ba210f3583a5ac0","reportUrl":"../2093//#testresult/ba210f3583a5ac0","status":"passed","time":{"start":1721439332096,"stop":1721439512024,"duration":179928}},{"uid":"3f320e4e936ae366","reportUrl":"../2087//#testresult/3f320e4e936ae366","status":"passed","time":{"start":1721356652345,"stop":1721356827568,"duration":175223}},{"uid":"23108d0101486446","reportUrl":"../2082//#testresult/23108d0101486446","status":"passed","time":{"start":1721266652203,"stop":1721266858655,"duration":206452}},{"uid":"5364b16e0bf261e1","reportUrl":"../2079//#testresult/5364b16e0bf261e1","status":"passed","time":{"start":1721180387447,"stop":1721180581896,"duration":194449}},{"uid":"7041bff13a8fc89d","reportUrl":"../2077//#testresult/7041bff13a8fc89d","status":"passed","time":{"start":1721093808909,"stop":1721093985926,"duration":177017}},{"uid":"5a59c85e1811c44f","reportUrl":"../2071//#testresult/5a59c85e1811c44f","status":"passed","time":{"start":1721007602430,"stop":1721007783462,"duration":181032}},{"uid":"8612636222601784","reportUrl":"../2070//#testresult/8612636222601784","status":"passed","time":{"start":1720921136201,"stop":1720921307841,"duration":171640}},{"uid":"79cb1171778e8c3a","reportUrl":"../2069//#testresult/79cb1171778e8c3a","status":"passed","time":{"start":1720834571772,"stop":1720834758241,"duration":186469}}]},"tags":["asyncio","abort_on_fail","@pytest.mark.skipif(False, reason='Requires amd64 architecture')","group(1)"]},"source":"d317f8a5d2f7d7ad.json","parameterValues":[]}